tests/unit/test_model.py

```diff
@@ -561,6 +561,20 @@ def test__IN():
         # Also verify the alias
         assert or_node == prop.IN(["a", None, "xy"])
 
+    @staticmethod
```

Comment:
Can we also have a version of `test__IN` that explicitly sets `force_server=False` rather than relying on the default value?

Reply:
I added an explicit test and renamed the value to server_op.  I'd be happy to pick another name.

Reply:
`server_op` is fine by me!